### PR TITLE
CSD-15 Opportunity results filter styling

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# NOT READY FOR REVIEW

# Summary
- Opportunity filter styling

# Needed By (Date)
- asap

# Urgency
- medium

# Steps to Test
1. Pull this branch
2. Checkout local Opportunity filter page

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [CSD-15](https://stanfordits.atlassian.net/browse/CSD-15)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by rebeccahongsf** [comment]
@pookmish I think is this almost ready for review but I was wonder in terms of the full width hero displayed within the [mock](https://www.figma.com/file/CpblbZldiGM6oYawa9aB4Q/Look-%26-Feel?node-id=380%3A2207) — is this something that we should add and configure for this page/layout or will we return to that later on down the road? 🤔

**Comment by pookmish** [comment]
Yeah i think since we have that layout with the overlay we can accomplish that without any additional development.

This PR does have a couple codeclimate issues and a merge conflict. The merge conflict can be fixed by merging in the latests `8.x-1.x` branch and then recompiling the css.

File: themes/cardinal_service/pdb_components/opportunities_list/src/Components/Filters.jsx [hunk 1]
```diff
@@ -1,13 +1,61 @@
...
+    justify-content: center;
+  }
+
+  @media (max-width: 768px) {
+    .flex-10-of-12,
+    .flex-2-of-12 {
+      flex: unset;
+      width: 100%;
+      max-width: unset;
+    }
```
Comment (pookmish) [inline themes/cardinal_service/pdb_components/opportunities_list/src/Components/Filters.jsx:24]: if you're changing their widths, you can use the classes `flex-md-10-of-12` or similar. available classes contain `sm`, `md`, `lg`, `xl` and `2xl`. see https://decanter.stanford.edu/page/layouts-grid-system/

Comment (rebeccahongsf) [reply themes/cardinal_service/pdb_components/opportunities_list/src/Components/Filters.jsx:24]: This is exactly what I needed! Thanks Mike — I'll update the classes

File: themes/cardinal_service/lib/scss/opportunities/_opportunities-filter.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+@charset 'UTF-8';
+
+.opportunity-filter-hero-container {
```
Comment (rebeccahongsf) [inline themes/cardinal_service/lib/scss/opportunities/_opportunities-filter.scss:3]: This (in addition to the Opportunity page hero styling) can be turned into a universal class.

File: themes/cardinal_service/lib/scss/opportunities/_opportunities-filter.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+.opportunity-filter-search-container {
+  background-color: #f4f4f4;
+  border-bottom: 1px solid #e5e5e5;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.25);
+  margin-bottom: 72px;
+
+  .opportunties-list {
+    @include centered-column;
+  }
+}
```
Comment (rebeccahongsf) [inline themes/cardinal_service/lib/scss/opportunities/_opportunities-filter.scss:26]: Currently this container styled the entire react components. However, the [mock](https://www.figma.com/file/CpblbZldiGM6oYawa9aB4Q/Look-%26-Feel?node-id=380%3A2207) up shows that the background only appears behind the SelectLists, excluding the Slugs.

Something we can review during our meeting after lunch tomorrow? 😄

Comment (rebeccahongsf) [reply themes/cardinal_service/lib/scss/opportunities/_opportunities-filter.scss]: After thinking and sleeping on this, I decided to move the styling into the React Components.

File: themes/cardinal_service/lib/scss/opportunities/_opportunities-filter.scss (hunk 2 above)
Comment (rebeccahongsf) [reply themes/cardinal_service/lib/scss/opportunities/_opportunities-filter.scss:3]: Now once I figure out how to make the hero styling a reusable classes, we can remove this `opportunity_filter.scss` file all together! 🤔

**Comment by rebeccahongsf** [comment]
**Need to update:**
- Move reusable classes into a utility partial file
- Delete `_opportunities-filter.scss`

**Pull request merged**